_src/static/sass/utilities/_variables.scss

```diff
@@ -96,14 +96,14 @@ $palettes: (
     grayscale:           (
         base:       $gray,
         mid-light:  tint($gray, 40%),
-        light:      tint($gray, 60%),
-        x-light:    tint($gray, 80%),
+        light:      tint($gray, 50%),
+        x-light:    tint($gray, 75%),
         xx-light:   tint($gray, 90%),
         white:      tint($gray, 98%),
         white-t:    tint($gray, 100%),
         mid-dark:   shade($gray, 40%),
-        dark:       shade($gray, 60%),
-        x-dark:     shade($gray, 80%),
+        dark:       shade($gray, 50%),
```

Comment:
Why were these changed? If they're to address a11y concerns, cool. If for some other reason, what was it?